R/connection.R

```diff
@@ -116,30 +116,32 @@ close_connection <- function(conn) {
 #'
 #' @template db_table_id
 #' @template conn
+#' @param allow_table_only
+#'  logical. If `TRUE`, allows for returning an `DBI::Id` with `table` = `myschema.table` if schema `myschema`
```

Comment:
Add that `TRUE` is default

Reply:
Is this really necessary? It is specified in the function call and therefore also shown on the help page:

`id(db_table_id, conn = NULL, allow_table_only = TRUE)`